src/common/snippets/src/pass/mha_tokenization.cpp

```diff
@@ -233,7 +234,7 @@ bool update_intermediate_supported_ops(std::shared_ptr<ov::Node>& interm_op,
 }
 
 std::vector<int32_t> get_rank_equivalent_order(std::vector<int32_t> default_order, size_t rank) {
-    assert(rank > 2 && "Incorrect order rank for Transpose tokenization");
+    OPENVINO_DEBUG_ASSERT(rank > 2, "Incorrect order rank for Transpose tokenization, expected rank > 2, got ", rank);
```

Comment:
If it works the same as the `OPENVINO_ASSERT` the condition `rank > 2` will be also put in error message so `expected rank > 2,` could be redundant.

Reply:
I agree, updated comment